fuseprivate.c

```diff
@@ -65,11 +65,14 @@ void sqfs_usage(char *progname, bool fuse_usage, bool ll_usage) {
 		progname ? progname : PACKAGE_NAME);
 	fprintf(stderr, "\n%s options:\n", progname);
 	fprintf(stderr, "    -o offset=N            offset N bytes into ARCHIVE to mount\n");
+	fprintf(stderr, "    -o subdir=/dir         mount subdirectory /dir of image\n");
+
 	if (ll_usage) {
 		fprintf(stderr, "    -o timeout=N           idle N seconds for automatic unmount\n");
-		fprintf(stderr, "    -o uid=N               set file owner to uid N\n");
-		fprintf(stderr, "    -o gid=N               set file group to gid N\n");
+		fprintf(stderr, "    -o uid=N               set file owner\n");
+		fprintf(stderr, "    -o gid=N               set file group\n");
```

Comment:
Please restore the original messages here.

Reply:
Done.